packages/world/src/ResourceSelector.sol

```diff
@@ -1,5 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
+import { ROOT_NAMESPACE, ROOT_FILE } from "./constants.sol";
+import { Bytes } from "@latticexyz/store/src/Bytes.sol";
+
+bytes16 constant ROOT_NAMESPACE_STRING = bytes16("ROOT_NAMESPACE");
+bytes16 constant ROOT_FILE_STRING = bytes16("ROOT_FILE");
```

Comment:
Do we have any guard rails to keep folks from registering using these constants?

Reply:
Not really, but also they're only used for logging, mostly in errors. Not worth the gas cost of adding checks to prevent them from being used as actual namespaces/files imo, wdyt?

Reply:
got it, I missed that this was only used in logging!